doc/man3/SSL_CTX_set_new_pending_conn_cb.pod

```diff
@@ -49,9 +49,9 @@ QUIC connections may begin processing prior to when an application calls
 SSL_accept_connection() on them.  As such, it may occur that callbacks are
 delivered to appliations' registered TLS callbacks prior to those SSL objects
 being returned in SSL_accept_connection().  Applications should expect this
-possibilty
+possibility
```

Comment:
There is another typo - `appliations'` -> `applications'`

Also missing a dot at the end of the paragraph.